### PR TITLE
Update quarkus-backed to the latest Quarkus version

### PR DESCRIPTION
**Pull request by iocanel**
## What does this PR do / why we need it

This pull request updates the Quarkus Backend template to latest Quarkus version

### CODE REVIEW

**Comment by iocanel** [comment]
Created the pull request by accident before its ready, so I am converting to draft.

**Review by kadel — APPROVED** [review]

File: templates/github/quarkus-backend/template.yaml [hunk 1]
```diff
@@ -58,7 +58,8 @@ spec:
       required:
         - groupId
         - artifactId
-        - javaPackageName
+        - version
+        - java_package_name
```
Comment (schultzp2020) [inline templates/github/quarkus-backend/template.yaml:62]:
```suggestion
        - javaPackageName
```

Please use camelcase.

File: templates/github/quarkus-backend/template.yaml [hunk 2]
```diff
@@ -70,11 +71,16 @@ spec:
           type: string
           default: quarkusapp
           description: Maven Artifact ID
-        javaPackageName:
+        version: 
+          title: Version
+          type: string
+          default: 1.0.0-SNAPSHOT
+          description: Maven Artifact Version
+        java_package_name:
```
Comment (schultzp2020) [inline templates/github/quarkus-backend/template.yaml:79]:
```suggestion
        javaPackageName:
```

File: templates/github/quarkus-backend/template.yaml [hunk 3]
```diff
@@ -143,7 +149,8 @@ spec:
           ci: ${{ parameters.ci }}
           groupId: ${{ parameters.groupId }}
           artifactId: ${{ parameters.artifactId }}
-          javaPackageName: ${{ parameters.javaPackageName }}
+          version: ${{ parameters.version }}
+          java_package_name: ${{ parameters.java_package_name }}
```
Comment (schultzp2020) [inline templates/github/quarkus-backend/template.yaml:153]:
```suggestion
          javaPackageName: ${{ parameters.javaPackageName }}
```

File: templates/github/quarkus-backend/template.yaml [hunk 4]
```diff
@@ -168,7 +175,7 @@ spec:
           ci: ${{ parameters.ci }}
           groupId: ${{ parameters.groupId }}
           artifactId: ${{ parameters.artifactId }}
-          javaPackageName: ${{ parameters.javaPackageName }}
+          java_package_name: ${{ parameters.java_package_name }}
```
Comment (schultzp2020) [inline templates/github/quarkus-backend/template.yaml:178]:
```suggestion
          javaPackageName: ${{ parameters.javaPackageName }}
```

File: templates/github/quarkus-backend/template.yaml [hunk 5]
```diff
@@ -191,7 +198,7 @@ spec:
...
```
Comment (schultzp2020) [inline templates/github/quarkus-backend/template.yaml:201]:
```suggestion
          javaPackageName: ${{ parameters.javaPackageName }}
```

**Review by schultzp2020 — CHANGES_REQUESTED** [review]
Don't forget to update `java_package_name` to `javaPackageName` throughout the skeleton.

**Comment by iocanel** [comment]
@schultzp2020: Thanks for the feedback. Updated the PR to use camel case everywhere.

**Review by schultzp2020 — APPROVED** [review]
LGTM

**Pull request merged**